templates/default/ckan_properties.ini.erb

```diff
@@ -237,7 +237,7 @@ ckan.storage_path = /var/shared_content/<%= @app_name %>/ckan_storage
 
 ckanext.s3filestore.aws_use_ami_role = true
 ckanext.s3filestore.aws_bucket_name = <%= node['datashades']['attachments_bucket'] %>
-ckanext.s3filestore.host_name = https://s3-ap-southeast-2.amazonaws.com
+ckanext.s3filestore.host_name = https://<%= @app_url %><% node['datashades']['ckan_web']['endpoint'] %><%= node['datashades']['attachments_bucket'] %>
```

Comment:
should this be a single variable injected instead of a build up inside the cookbook?

Reply:
There isn't anywhere else that uses the same combination of site URL and bucket name, so probably not.